index.js

```diff
@@ -55,3 +56,20 @@ nextButton.addEventListener("click", () => {
 function updatePagination() {
   pagination.textContent = `${page} / ${maxPage}`;
 }
+
+searchBar.addEventListener("submit", (event) => {
+  event.preventDefault();
+  const formData = new FormData(searchBar);
+  searchQuery = formData.get("query").trim();
+  page = 1;
+  fetchCharacters();
+});
+
+searchInput.addEventListener("input", (event) => {
```

Comment:
I guess the eventListener "input" is needed to return to the default state of the page when a user clears the input-field. Cool idea! Would there be another purpose why we implement the input-EventListener? -> we can discuss tomorrow;)

Reply:
Yes, i created the "input" eventListener, so when the user clear search field, first site loaded automatically.